applications/GeoMechanicsApplication/tests/cpp_tests/custom_geometries/test_interface_geometry.cpp

```diff
@@ -376,7 +376,7 @@ KRATOS_TEST_CASE_IN_SUITE(InterfaceGeometry_ReturnsCorrectJacobian_ForTwoPlusTwo
 
     Matrix expected_result(2, 1);
     expected_result <<= 3.25, 0.0;
-    KRATOS_EXPECT_MATRIX_RELATIVE_NEAR(result, expected_result, Defaults::relative_tolerance)
+    KRATOS_EXPECT_MATRIX_RELATIVE_NEAR(result, expected_result, Defaults::absolute_tolerance)
```

Comment:
More nagging: this check assumes a relative tolerance, but receives an absolute one:
```suggestion
    KRATOS_EXPECT_MATRIX_RELATIVE_NEAR(result, expected_result, Defaults::arelative_tolerance)
```

Reply:
Whoops, my mistake, corrected